test/index.test.ts

```diff
@@ -1,7 +1,52 @@
-import helloWorld from '../src/index';
+// Imports
+import cssValidator from '../src';
 
-describe('#helloWorld()', () => {
-	it('Works', () => {
-		expect(helloWorld()).toEqual('Hello world!');
+// Tests
+describe('#validateText()', () => {
+	afterEach(
+		() => new Promise<void>((resolve) => setTimeout(resolve, 1000))
```

Comment:
We need to explain why we're doing this. Probably quote the page, and link to the page it comes from.